.gitignore

```diff
@@ -75,3 +75,5 @@ tags
 .vscode/*
 .history
 # End of https://www.gitignore.io/api/go,vim,emacs,visualstudiocode
+
+vendor/
```

Comment:
Right now we have `vendor/` checked in to the repo. We should either remove this _and_ `vendor/` or include vendor.

Reply:
Thanks for pointing that out. I will remove this line for now and do a follow up PR that removes `vendor/` + adds this line in the `.gitignore` file + sorts out some dependency management related confusion